Source/Core/Core/HotkeyManager.cpp

```diff
@@ -290,7 +291,7 @@ constexpr std::array<HotkeyGroupInfo, NUM_HOTKEY_GROUPS> s_groups_info = {
      {_trans("Controller Profile"), HK_NEXT_WIIMOTE_PROFILE_1, HK_PREV_GAME_WIIMOTE_PROFILE_4},
      {_trans("Graphics Toggles"), HK_TOGGLE_CROP, HK_TOGGLE_TEXTURES},
      {_trans("Internal Resolution"), HK_INCREASE_IR, HK_DECREASE_IR},
-     {_trans("Freelook"), HK_FREELOOK_DECREASE_SPEED, HK_FREELOOK_RESET},
```

Comment:
You removed the reset hotkey instead of adding a toggle hotkey here.

Reply:
No this is correct, it sets up the hotkey ranges for the tabs. When a new hotkey is added after the end of a range you have to change the end of the range to include the new hotkey.

Reply:
Oh ok, it certainly looks like a removal here. Consider that revolved then!